src/service-worker.js

```diff
@@ -417,11 +417,7 @@ export const passThrough = (event, te, oUrl) => {
 
   // default: pass requests through to the network
   // console.log('-- (rtr-sw) passThrough NON-OKAPI', req.url)
-  return fetch(event.request, { credentials: 'include' })
-    .catch(e => {
-      console.error(e); // eslint-disable-line no-console
-      return Promise.reject(new Error(e));
-    });
+  return fetch(event.request);
```

Comment:
Just checking: was it deliberate that as well as taking out this catch clause you also removed `credentials: 'include'`?

Reply:
It was, based on [examples from the documentation](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent). Looking at that example again, I wonder if an _even better_ implementation here would be to do some of the request-sniffing that currently happens in `passThrough` in the event handler instead, returning immediately for requests that happen outside the RTR workflow, before calling `event.respondWith(...)`.

Thank you for mentioning this. I will investigate further.